internal/cli/up.go

```diff
@@ -96,9 +96,15 @@ func (c *upCmd) run(ctx context.Context, args []string) error {
 		return err
 	}
 
+	l := upper.LogActionLogger(ctx)
+	origCtx := ctx
+	ctx = logger.WithLogger(ctx, l)
```

Comment:
or... call this `ctxWithLogger` and only use it in the one place? won't this potentially do weird things to the code below if we're mucking with the value of `ctx` in a goroutine?

Reply:
yeah, fair

This comment also made me realize that it was a bit dangerous to just slurp all of our logs into the hud logger - if there's a problem with the hud, we won't see those logs.